proposals/0107-unsaferawpointer.md

```diff
@@ -672,8 +670,8 @@ pointer cast is explicit:
 ```swift
 // --- new version ---
 func testInitAorB() {
-  let p = UnsafeMutableRawPointer.allocate(bytes: sizeof(Int.self),
-                                           alignedTo: alignof(Int.self))
+  let p = UnsafeMutableRawPointer.allocate(bytes: MemoryLayout<Int>.stride,
+                                           alignedTo: MemoryLayout<Int>.alignment)
 
   initRawAorB(p, isA: true)
   printA(p.assumingMemoryBound(to: A.self))
```

Comment:
It's ok to reinitialize trivial types. It doesn't even matter here that they are layout compatible (as long as the allocated memory is large enough and properly aligned). I think there was initially some concern about exposing the term "trivial type" in these doc comments. But that cat is out of the bag.

I'll fix the doc comments:
https://github.com/apple/swift/pull/4560